src/packages/log-viewer/workspace/views/overview/components/log-viewer-saved-searches-overview.element.ts

```diff
@@ -3,12 +3,25 @@ import { UmbTextStyles } from "@umbraco-cms/backoffice/style";
 import { css, html, customElement, state } from '@umbraco-cms/backoffice/external/lit';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
 import { SavedLogSearchResponseModel } from '@umbraco-cms/backoffice/backend-api';
+import {debounceTime, Subject, tap} from "@umbraco-cms/backoffice/external/rxjs";
+import {path, query as getQuery, toQueryString} from "@umbraco-cms/backoffice/external/router-slot";
 
 //TODO: implement the saved searches pagination when the API total bug is fixed
 @customElement('umb-log-viewer-saved-searches-overview')
 export class UmbLogViewerSavedSearchesOverviewElement extends UmbLitElement {
 	@state()
 	private _savedSearches: SavedLogSearchResponseModel[] = [];
+	@state()
+	private _inputQuery = '';
+
+	@state()
+	private _showLoader = false;
+
+	@state()
+	private _isQuerySaved = false;
```

Comment:
I can only see that his property along with `_showLoader` gets written to but not read. Is it perhaps some kind of leftover from something else?

Reply:
Yes, you're right, removing that state parameter.